docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst

```diff
@@ -0,0 +1,52 @@
+.. _pro-check-list-cves:
+
+Given a list of CVEs, how to know if you are affected by them?
+**************************************************************
+
+If you have a list of Common Vulnerabilities and Exposures (CVE) or Ubuntu Security Notifications (USN) and want to check if your Ubuntu system is affected by them, you can easily check using the pro fix API. In this guide we'll show you how to leverage the pro fix API:
```

Comment:
We probably want to include a link to the page that describes CVEs (this might be a follow-up PR after the security CVE page changes land - I'm not sure of the URL yet) or to our explanation page explaining CVEs/USNs

Reply:
There are already security pages linked for CVEs and USNs in a different howto guide, I can just use those and link them here as well.